plugins/pulp_docker/plugins/distributors/configuration.py

```diff
@@ -232,6 +232,24 @@ def get_repo_relative_path(repo, config):
     return repo.id
 
 
+def get_remote_repo_relative_path(repo, config):
+    """
+    Get the configured relative path for the given repository. This method is used by rsync
+    distributor.
+
+    :param repo: repository to get relative path for
+    :type  repo: pulp.plugins.model.Repository
+    :param config: configuration instance for the repository
+    :type  config: pulp.plugins.config.PluginCallConfiguration or dict
+    :return: relative path for the repository
+    :rtype:  str
+    """
+    repo_relative_path = config.get('repo_relative_path', None)
```

Comment:
I updated it just use repo.id as the default. It's a repository mongoengine model. repo.id is correct.